src/main/java/lotto/domain/RankCount.java

```diff
@@ -0,0 +1,65 @@
+package lotto.domain;
+
+import java.util.EnumMap;
+
+import static lotto.domain.Rank.*;
+
+public class RankCount {
+    private static final int ZERO = 0;
+
+    private EnumMap<Rank, Integer> rankCount = new EnumMap<>(Rank.class);
+
+    public RankCount() {
+        rankCount.put(FIRST, 0);
+        rankCount.put(SECOND, 0);
+        rankCount.put(THIRD, 0);
+        rankCount.put(FOURTH, 0);
+        rankCount.put(FIFTH, 0);
+        rankCount.put(MISS, 0);
```

Comment:
Rank 이넘의 값이 추가되거나 삭제될때마다 초기화 로직도 수정해줘야하네요!
아래처럼 작성하면 초기화 로직은 영향이 없을 것 같습니다.
```suggestion
	Arrays.stream(values())
		.forEach(rank -> rankCount.put(rank, 0));
```